package.json

```diff
@@ -13,6 +13,7 @@
     "types/",
     "android/src",
     "android/build.gradle",
+    "android/testresults.gradle",
```

Comment:
@Krizzu: Why is this published? Shouldn't the script just not apply this file?

Reply:
Yeah, that'd be a follow up PR to apply this conditionally

Reply:
Or maybe we could just inline it